internal/service/globalclusterconfig/resource_global_cluster_config_test.go

```diff
@@ -16,35 +16,37 @@ import (
 )
 
 func TestAccClusterRSGlobalCluster_basic(t *testing.T) {
+	acc.SkipTestForCI(t) // needs to be fixed: "cloud_backup": conflicts with backup_enabled
```

Comment:
Could you create a  CLOUDP and add it to this comment? this would help us to track the work needed

Reply:
sure but let me first take a look at all Skip tests to see if they could be in the Test project or they can be non-epic stories